### PR TITLE
Add validation for days of the week.

### PR DESCRIPTION
**Pull request by kleschenko**

### CODE REVIEW

File: pendulum/pendulum.py [hunk 1]
```diff
@@ -787,6 +789,8 @@ def set_week_ends_at(cls, value):
 
         :type value: int
         """
+        if value not in range(7):
```
Comment (sdispater) [inline pendulum/pendulum.py]: I would do:

```python
if value not in cls._days:
```

File: pendulum/pendulum.py [hunk 2]
```diff
@@ -787,6 +789,8 @@ def set_week_ends_at(cls, value):
...
+            raise ValueError('Invalid day of the week value: {}'.format(value))
```
Comment (sdispater) [inline pendulum/pendulum.py]: The message could be just:

```python
'Invalid day of the week: {}'.format(value)
```

File: pendulum/pendulum.py (hunk 1 above)
Comment (kleschenko) [inline pendulum/pendulum.py]: Updated.

**Comment by sdispater** [comment]
Thanks!

**Pull request merged**